ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageManager.java

```diff
@@ -92,6 +96,23 @@ public void setTintColor(ReactImageView view, @Nullable Integer tintColor) {
     }
   }
 
+  @ReactProp(name = "loadHandlersRegistered")
```

Comment:
I'd prefer for this property to be called "shouldNotifyLoadEvents" or similar instead "loadHandlersRegistered"